docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+API Reference
+=============
+
+Module contents
+---------------
+
+.. automodule:: scrapinghub.client
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Submodules
+----------
+
+activity
+--------
+
+.. automodule:: scrapinghub.client.activity
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
you need to add :inherited-members: everywhere because otherwise methods like `list` are not listed in the docs (http://stackoverflow.com/questions/30344289/how-can-i-show-inherited-members-of-a-class-in-my-sphinx-documentation)

<img width="409" alt="api_reference_ _scrapinghub_2_0_0_dev0_documentation" src="https://cloud.githubusercontent.com/assets/744331/24370122/28b0eb44-131e-11e7-9fef-9aef96e5fb93.png">